### ISSUE TITLE
Can't we use sort_by parameter

### ISSUE THREAD

**Issue by prono69** [sskender/pornhub-api#12]
We can search but can we use sort_by="mostviewd" or sort_by="newest" parameter ?
Currently it's using the default parameter right ?

**Comment by sskender** [comment]
Yes, currently it is using the default parameter. Advanced sort is not implemented. If you have time and energy to implement it, I will be happy to merge.

**Comment by sskender** [comment]
Fixed by @SashaSZ
